### DISCUSSION TITLE
Bootstrap dowload run npm install ater this run npm start get that error of hugo-bin

### DISCUSSION THREAD

**Discussion by shersial** [twbs/bootstrap#35538] [General] [answered]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Error: spawn C:\_mywork\bootstrap-5.0.2\node_modules\hugo-bin\vendor\hugo.exe ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)     
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (node:internal/child_process:288:12)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\_mywork\\bootstrap-5.0.2\\node_modules\\hugo-bin\\vendor\\hugo.exe',
  path: 'C:\\_mywork\\bootstrap-5.0.2\\node_modules\\hugo-bin\\vendor\\hugo.exe',
  spawnargs: [ 'server', '--port', '9001', '--disableFastRender' ]

### Reduced test cases

"name": "bootstrap",
    "version": "5.0.2",
- Clone the repo: `git clone https://github.com/twbs/bootstrap.git`
- Install with [npm](https://www.npmjs.com/): `npm install bootstrap`
- Install with [yarn](https://yarnpkg.com/): `yarn add bootstrap`
after npm start get error

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Bootstrap are you using?

5.0.2

**Comment by ffoodd** [comment, marked as answer]
You're trying to run our docs environment, is that what you want?

**Reply by shersial** [reply]
Yes 

1. download latest from https://github.com/twbs/bootstrap/archive/v5.1.3.zip
2. npm install bootstrap
3. Run npm install
4. Run npm run test
5. get that error
Chrome Headless 96.0.4664.93 (Windows 10): Executed 697 of 697 SUCCESS (22.117 secs / 21.589 secs)
TOTAL: 697 SUCCESS

=============================== Coverage summary ===============================
Statements   : 96.95% ( 2040/2104 )
Branches     : 90.89% ( 1048/1153 )
Functions    : 98.09% ( 411/419 )
Lines        : 96.91% ( 1980/2043 )
================================================================================

> bootstrap@5.1.3 docs-build
> hugo --cleanDestinationDir

node:events:368
      throw er; // Unhandled 'error' event
      ^

Error: spawn C:\_mywork\bootstrap-5.1.3\bootstrap-5.1.3\node_modules\hugo-bin\vendor\hugo.exe ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (node:internal/child_process:288:12)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\_mywork\\bootstrap-5.1.3\\bootstrap-5.1.3\\node_modules\\hugo-bin\\vendor\\hugo.exe',
  path: 'C:\\_mywork\\bootstrap-5.1.3\\bootstrap-5.1.3\\node_modules\\hugo-bin\\vendor\\hugo.exe',
  spawnargs: [ '--cleanDestinationDir' ]
}
ERROR: "docs-build" exited with 1.

**Reply by ffoodd** [reply]
What version of Node are you using? FWIW that's a Hugo issue, you may find something useful in their repo.

**Reply by Fearless09** [reply]
I'm sorry, I don't mean to be rude but I don't see how this answer the question.

I'm also having the same problem and yet to find solution to it

**Reply by Fearless09** [reply]
I'll be glad if you can help me out

**Comment by XhmikosR** [comment]
1. v5.0.2 is severely outdated
2. hugo-bin has trouble finding the binary on your system for some unknown reason; could be a network issue, an antivirus or something else

**Reply by shersial** [reply]
download latest from https://github.com/twbs/bootstrap/archive/v5.1.3.zip
npm install bootstrap
Run npm install
Run npm run test
get that error
Chrome Headless 96.0.4664.93 (Windows 10): Executed 697 of 697 SUCCESS (22.117 secs / 21.589 secs)
TOTAL: 697 SUCCESS
=============================== Coverage summary ===============================
Statements : 96.95% ( 2040/2104 )
Branches : 90.89% ( 1048/1153 )
Functions : 98.09% ( 411/419 )
Lines : 96.91% ( 1980/2043 )
bootstrap@5.1.3 docs-build
hugo --cleanDestinationDir

node:events:368
throw er; // Unhandled 'error' event
^

Error: spawn C:_mywork\bootstrap-5.1.3\bootstrap-5.1.3\node_modules\hugo-bin\vendor\hugo.exe ENOENT
at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
at onErrorNT (node:internal/child_process:477:16)
at processTicksAndRejections (node:internal/process/task_queues:83:21)
Emitted 'error' event on ChildProcess instance at:
at Process.ChildProcess._handle.onexit (node:internal/child_process:288:12)
at onErrorNT (node:internal/child_process:477:16)
at processTicksAndRejections (node:internal/process/task_queues:83:21) {
errno: -4058,
code: 'ENOENT',
syscall: 'spawn C:\_mywork\bootstrap-5.1.3\bootstrap-5.1.3\node_modules\hugo-bin\vendor\hugo.exe',
path: 'C:\_mywork\bootstrap-5.1.3\bootstrap-5.1.3\node_modules\hugo-bin\vendor\hugo.exe',
spawnargs: [ '--cleanDestinationDir' ]
}
ERROR: "docs-build" exited with 1.

**Comment by shersial** [comment]
PS C:\Windows\System32> node
Welcome to Node.js v16.13.1.
Type ".help" for more information.
>
(To exit, press Ctrl+C again or Ctrl+D or type .exit)
>
PS C:\Windows\System32> npm --version
8.1.1
PS C:\Windows\System32>

**Comment by shersial** [comment]
PS C:\Windows\System32> node
Welcome to Node.js v16.13.1.
Type ".help" for more information.
>
(To exit, press Ctrl+C again or Ctrl+D or type .exit)
>
PS C:\Windows\System32> npm --version
8.1.1
PS C:\Windows\System32>


M Zahid Sher Sial
Mobile: 0097 158 5879486
Mobile: 0092 336 4199486
Skype: zahesial
Email:  ***@***.***
Design Portfolio: www.behance.net/shersials
<http://shersial.com/>
Portfolio: https://www.behance.net/agileleaf



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
12/15/21,
09:59:27 AM

**Reply by potatokillar** [reply]
hi shersial,I have the same issue. so how to resolve this question?
